src/implementations/vanilla/components/select/_select.scss

```diff
@@ -100,7 +125,9 @@ $_arrow-width: 2.75rem;
 
 .ecl-select__icon {
   align-items: center;
-  background-color: $_border-color;
+  background-color: $_arrow-background-color;
+  border-bottom-right-radius: $_border-radius;
```

Comment:
Isn't it easier setting the shortcut border-radius..? One line instead of two, but just a thing i've noticed, nothing wrong with this..

Reply:
no otherwise we would have rounded edges on the left, and the caret button only has one on the right for EU.